### PR TITLE
Refactor itinerary

### PR DESCRIPTION
**Pull request by Tuxosaurus**
Refactoring:
- clean CSS
- remove pseudo-elements
- create "Bullet" component

Only visual change is the "Bullet" size is bigger, as intended in the design

### CODE REVIEW

File: src/bullet/index.tsx [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    box-sizing: content-box;
+    width: 6px;
+    height: 6px;
+    background-color: ${color.white};
+    border: 2px solid ${color.primaryText};
+    border-radius: 50%;
+  }
+
+  &.kirk-bullet--addon {
+    border-color: ${color.border};
```
Comment (pith) [inline src/bullet/index.tsx]: Not sure we want to use the border semantic here.

File: src/bullet/index.tsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+  }
+
+  &.kirk-bullet--small {
+    width: 4px;
+    height: 4px;
+  }
+
+  &.kirk-bullet--map {
```
Comment (pith) [inline src/bullet/index.tsx]: I'm not sure map is really clear here. As it will not be used in the map but in the itinerary component

Comment (pith) [reply src/bullet/index.tsx]: It's more a highlight WDYT? 🤷‍♂

Comment (Tuxosaurus) [reply src/bullet/index.tsx]: @mcellier actually requested to have a separate component so he could use it on maps too

File: src/bullet/index.tsx (hunk 1 above)
Comment (Tuxosaurus) [reply src/bullet/index.tsx]: Yes, I only moved what was already there. Would `color.fadedText` be OK? It's actually used along with an actual faded text

File: src/bullet/index.tsx (hunk 2 above)
Comment (mcellier) [reply src/bullet/index.tsx]: Yes, I did!

File: src/bullet/Bullet.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+  ADDON = 'addon',
+  MAP = 'map',
+}
+
+export interface BulletProps {
+  readonly className?: Classcat.Class
+  readonly type?: BulletTypes
+}
+
+const Bullet = ({ className, type }: BulletProps) => {
```
Comment (mcellier) [inline src/bullet/Bullet.tsx]: Use defaultProps instead of isEmpty(type)

**Pull request merged**